cli/stat_test.go

```diff
@@ -0,0 +1,160 @@
+package cli_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/coder/coder/cli/clitest"
+	"github.com/coder/coder/testutil"
+)
+
+// This just tests that the stat command is recognized and does not output
+// an empty string. Actually testing the output of the stats command is
+// fraught with all sorts of fun. Some more detailed testing of the stats
+// output is performed in the tests in the clistat package.
+func TestStatCmd(t *testing.T) {
+	t.Parallel()
+	t.Run("JSON", func(t *testing.T) {
+		t.Parallel()
+		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitShort)
+		t.Cleanup(cancel)
+		inv, _ := clitest.New(t, "stat", "all", "--output=json")
```

Comment:
Should we add golden files for json output?

Reply:
They'll be different all the time!

Reply:
If the intent is to ensure we don't break the output, we could instead just assert non-zero values in the JSON. I don't consider the addition of fields to be a breaking change.